### PR TITLE
add the list and cancel database operations

### PR DESCRIPTION
**Pull request by jiawang003**
add the list and cancel database operations under 2017-03-01-preview, there's no issue in autorest validation

### CODE REVIEW

**Comment by olydis** [comment]
@ravbhatnagar new APIs

**Comment by ravbhatnagar** [comment]
Other than that it looks fine.

File: specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/cancelOperations.json [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+                    "format": "uuid"
+                },
+                {
+                    "$ref": "#/parameters/SubscriptionIdParameter"
+                },
+                {
+                    "$ref": "#/parameters/ApiVersionParameter"
+                }
+            ],
+            "responses": {
```
[the remark this answers is not in the document]
Comment (jiawang003) [reply specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/cancelOperations.json]: yes it'll be updated as CANCELLED

File: specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/cancelOperations.json [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/operations/{operationId}/cancel": {
+        "post": {
+            "tags": [
+                "Databases"
+            ],
+            "description": "Cancels the asynchronous operation on the database.",
+            "operationId": "Databases_Cancel",
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/cancelOperations.json]: DatabaseOperations_Cancel

File: specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/cancelOperations.json [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+            }
+        }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/operations": {
+      "get": {
+        "tags": [
+          "Databases"
+        ],
+        "description": "Gets a list of operations performed on the database.",
+        "operationId": "Databases_ListOperations",
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/cancelOperations.json]: DatabaseOperations_ListByDatabase

**Review by jaredmoo — CHANGES_REQUESTED** [review]

**Comment by olydis** [comment]
@sergey-shandar Swapping this with you since I took the other one, let me know if that doesn't work for you 😉

**Pull request closed without merging**